spa-dashboard-angular/package.json

```diff
@@ -25,7 +25,7 @@
     "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^14.2.1",
+    "@angular-devkit/build-angular": "^17.3.4",
```

Comment:
They mayor version Angular version of this project is 14.
All Angular components should be updated together.